meeting-minutes/client-minutes/nov10-client-minutes.md

```diff
@@ -0,0 +1,83 @@
+- A road block from this week was the complexity that matching based on common availability introduces. Is common availability **required** when it comes to making mentee and mentor matches, or can we match primarily based on skills and time-zones?
+  - The client recognizes the complexity that matching based on common availability might have. As such, when creating mentee and mentor matches, **common availability does not need to be considered**.
+    - The client's recommended course of action: The process can be analogous to a job recruiting process (i.e The individual conducting interviews sends out a calendar to the interviewee. The interviewee receives a calendar of available time slots, and picks one). Therefore, for our project we can:
+      - Obtain availability for Mentors only.
+      - Once a match has been made, send the Mentor's availability to the Mentee.
+      - The mentee can then select a timeslot based on when the Mentor is available.
+
+The client asked how our team was progressing with our use of Firebase, so the MentorshipForm.js file was shown since authentication was worked on over the last sprint. The client recommended also initializing Firebase within App.js as well (so that all the components being called from App.js have access to the information in the database).
+
+The client also mentioned that we should keep in mind that the **Mentee** form will be embedded onto a public website, so we should consider how it will be turned into an iFrame tag.
+
+### ACTION ITEMS
+
+11/10-01. Re-vamp the Mentee and Mentor forms to include the input fields as listed. Since Eric was working on separating the MentorshipForm.js into a form for Mentees in specific, and another for Mentors, he could potentially refine the fields for each. **Should be completed by the end of this week's sprint [Monday]**
+11/10-02. The team should decide whether we want to remove the time-zone dropdown and replace it with obtaining a user's location. Potentially, we could keep the time-zone dropdown and add a field to obtain the user's location. We could present this during next week's client meeting, and inform the client of associated pros and cons to determine if we replace time-zone with user location. **This should be discussed with the team, and decided on by Wednesday so we can incorporate changes if necessary**
+11/10-03. The team should discuss possible ways of obtaining Mentor availability to send to an accepted Mentee, where the Mentee can then book a slot from the Mentor's available time slots. **Ideally, the team will brainstorm some ideas and come up with a solution that can be implemented by Thursday morning at the latest**
```

Comment:
Does the way I suggested work?

Reply:
Do you mean with having a date and time picker? I think that should work, however, I've been having issues when it comes to installing libraries due to our version of react (that, or with our version of npm). It looks like something along the lines of [this](https://www.npmjs.com/package/react-datetime) library is what we'd need, but I'm not able to install it successfully. 
Are you able to, or is there something else that would work better? For reference, this is the error I get:
<img width="622" alt="Screen Shot 2020-11-11 at 11 31 33 AM" src="https://user-images.githubusercontent.com/20407302/98855799-74f21c00-2411-11eb-98db-681bf7e775d3.png">
Luckily, we only need to worry about this for the Mentor form, and then I guess we'd just send the mentor's availability in the email which notifies mentees that they've been matched. For our current iteration, this would probably work. But in the long-term, we'd also need to consider how to block-off the selected time once booked (i.e If I'm a mentor, and I'm available Monday from 9-10AM, I match with a mentee, and they're available at that time as well so we decide to meet at that time, if a Mentor accepts another Mentee, the Monday 9-10AM time slot should no longer be available).
The simplest solution that I can see for this would be to setup something with Google Calendar, but I'm not sure if I'm over-thinking this.